app/mqtt.mjs

```diff
@@ -1,15 +1,264 @@
+        'overPressureTimeLeft': createGenericSensorConfiguration(configurationBase, 'overPressureTimeLeft', 'Overpressure time left', 'minutes'),
+        'ventilationLevelTarget': createGenericSensorConfiguration(configurationBase, 'ventilationLevelTarget', 'Ventilation level (target)', '%'),
+        'ventilationLevelActual': createGenericSensorConfiguration(configurationBase, 'ventilationLevelActual', 'Ventilation level (actual)', '%'),
+    }
+
+    // Configurable numbers
+    const numberConfigurationMap = {
+        'overPressureDelay': createNumberConfiguration(configurationBase, 'overPressureDelay', 'Overpressure delay', {
+            'min': 1,
+            'max': 60,
+            'unit_of_measurement': 'minutes',
+        }),
+        'awayVentilationLevel': createNumberConfiguration(configurationBase, 'awayVentilationLevel', 'Away ventilation level', {
+            'min': 1,
+            'max': 100,
+            'unit_of_measurement': '%',
+        }),
+        'awayTemperatureReduction': createNumberConfiguration(configurationBase, 'awayTemperatureReduction', 'Away temperature reduction', {
+            'min': 0,
+            'max': 20,
+            'unit_of_measurement': '°C',
+        }),
+        'longAwayVentilationLevel': createNumberConfiguration(configurationBase, 'longAwayVentilationLevel', 'Long away ventilation level', {
+            'min': 1,
+            'max': 100,
+            'unit_of_measurement': '%',
+        }),
+        'longAwayTemperatureReduction': createNumberConfiguration(configurationBase, 'longAwayTemperatureReduction', 'Long away temperature reduction', {
+            'min': 0,
+            'max': 20,
+            'unit_of_measurement': '°C',
+        }),
+        'temperatureTarget': createNumberConfiguration(configurationBase, 'temperatureTarget', 'Temperature target', {
+            'min': 0,
+            'max': 30,
+            'unit_of_measurement': '°C',
+        })
+    }
+
+    // Configurable switches
+    const switchConfigurationMap = {
+        'away': createSwitchConfiguration(configurationBase, 'away', 'Away'),
+        'longAway': createSwitchConfiguration(configurationBase, 'longAway', 'Long away'),
+        'overPressure': createSwitchConfiguration(configurationBase, 'overPressure', 'Overpressure'),
+        'maxHeating': createSwitchConfiguration(configurationBase, 'maxHeating', 'Max heating'),
+        'maxCooling': createSwitchConfiguration(configurationBase, 'maxCooling', 'Max cooling'),
+        'manualBoost': createSwitchConfiguration(configurationBase, 'manualBoost', 'Manual boost'),
+        'summerNightCooling': createSwitchConfiguration(configurationBase, 'summerNightCooling', 'Summer night cooling'),
+    }
+
+    // Publish configurations
+    for (const [entityName, configuration] of Object.entries(sensorConfigurationMap)) {
+        const configurationTopicName = `homeassistant/sensor/${deviceIdentifier}/${entityName}/config`
```

Comment:
Could use a templated const for topic here

Reply:
I'll refactor the configuration publishing to use a deeper map and fewer loops

Reply:
Refactored